packages/manager/modules/manager-pci-common/src/contexts/MetaContext.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+
+interface InternalMeta {
+  has3AZ?: boolean;
+}
+
+export type MetaType = InternalMeta | Record<string, unknown> | undefined;
+
+export const MetaContext = createContext<MetaType>(undefined);
```

Comment:
As the meta context provider is localized in the `RegionSelector` component, I will be more precise: 
```suggestion
export const RegionMetaContext = createContext<RegionMetaType>(undefined);
```

Reply:
@fredericvilcot as meta could be shared between multiple components I think we should keep it not localized

Or else we should rename meta prop to regionMeta in component and put relative files inside the region selector folder

Both solutions are good to me

Reply:
I agree to rename `RegionContext` as we put it in the folder `contexts`. If we need another context we will create another file. And I think also it's more clear to rename meta prop to regionMeta

Reply:
@SimonChaumet 
If we want to handle an agnostic object as `meta`, we could imagine removing the provider from the lib and then just consume the abstraction of the whole object thanks to context consumers